apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -6,6 +6,7 @@ import { DailyLocationType } from "@calcom/app-store/locations";
 import { getDownloadLinkOfCalVideoByRecordingId } from "@calcom/core/videoClient";
 import { sendDailyVideoRecordingEmails } from "@calcom/emails";
 import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
+import { getTeamId } from "@calcom/features/bookings/lib/handleNewBooking";
```

Comment:
nit: instead of importing getTeamId from handleNewBooking. It should be a utility in it's own maybe at packages/lib/getTeamId

Reply:
Yeah I also thought about this approach of a utility just wasn't sure about it, will. make the change.